src/components/StudentProfile.tsx

```diff
@@ -43,7 +43,7 @@ const StudentProfile: React.FC<StudentProfileProps> = ({ selectedStudent }) => {
         >
           {selectedStudent?.studentInfo.studentName}
         </Heading>
-        {selectedStudent?.studentInfo.selectedFact && selectedStudent?.studentInfo.selectedFact.value && (
+        {selectedStudent && selectedStudent.studentInfo?.selectedFact && (
```

Comment:
why abandon optional chaining?

Reply:
I have no idea why I did this.  I'll change it back

Reply:
I think it's cause further down you still wanted reference to this variable by itself or something?  idk, this is nitpicky so if you have a good reason for it and don't feel like refactoring it's fine but it looks a bit odd

Reply:
Looking at it in the file I don't think there's good reason for it.  I changed it back